includes/definitions/mrv-optidriver.yaml

```diff
@@ -0,0 +1,10 @@
+os: mrv-optidriver
+group: mrv
+text: 'MRV OptiDriver'
+type: network
+icon: mrv
+discovery:
+    - sysObjectId:
+       - .1.3.6.1.4.1.629.200.1.1.1.
```

Comment:
Looking at our stats, nothing else uses it so yes it should be ok to be shortened.